PhysicsTools/JetMCAlgos/plugins/HadronAndPartonSelector.cc

```diff
@@ -254,7 +254,7 @@ HadronAndPartonSelector::produce(edm::Event& iEvent, const edm::EventSetup& iSet
      partonSelector_->run(particles,partons);
      for(reco::GenParticleCollection::const_iterator it = particles->begin(); it != particles->end(); ++it)
      {
-       if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
+      // if( !(it->status()==3 || (( partonMode_=="Pythia8" ) && (it->status()==23)))) continue;
```

Comment:
Dear @Andrej-CMS,

I am wondering if commenting out this has any consequences not only in terms of file size but also in other places where HadronAndSelector is being used. 
It is because if I understand correctly, without this condition, this will contain all partons including status 2 and also stable final particles. That will be a lot of particles. This will also introduce ambiguity issue.

Best Regards,
Taejeong